src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -168,6 +177,51 @@ private static ScriptHostConfiguration GetScriptHostConfiguration(string scriptP
             return scriptHostConfig;
         }
 
+        private static string MakeValidHostId(string id)
```

Comment:
FWIW, this logic was already there, I just factored it into a separate method and made it more robust.